### PR TITLE
[ASM] - Add Email Notification for Remediation Action for Service Owners - Expander - 3032

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-3032](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3032)

## Description
Update the Cortex ASM - ASM Alert playbook with a task at the end of the "Remediation Stage" to send a remediation notification email to service owners.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by BigEasyJ** [comment]
Please do not merge until review/acceptance from @johnnywilkes or @capanw.

**Comment by daryakoval** [comment]
@BigEasyJ Hi, Is the pr ready for @capanw and @johnnywilkes review?

**Review by capanw — APPROVED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 1]
```diff
@@ -188,26 +188,27 @@ tasks:
     isautoswitchedtoquietmode: false
   "11":
     id: "11"
-    taskid: 97a8b99b-265b-493a-8a3c-5f22b17f5fc2
-    type: title
+    taskid: 322faa9a-3a45-476d-88ac-ba4cf2bcc9d5
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:191]: Please edit playbook so:
- email should only be sent for full AR
- Also, you need to add a conditional to check to make sure the service owner has an email.

**Review by johnnywilkes — CHANGES_REQUESTED** [review]

**Review by daryakoval — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@daryakoval , we are still waiting on a few changes

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 2]
```diff
@@ -2959,7 +2959,105 @@ tasks:
...
+              </p>
+              <p>
+                  <b>Alert Details:</b> ${alert.details}<br>
+                  <b>Action Taken:</b> ${alert.asmremediation.[0].Action}<br>
+                  <b>Action Outcome:</b> ${alert.asmremediation.[0].Outcome}<br>
+              </p>
+          </body>
+          </html>
+      subject:
+        simple: '[TESTING] A new security risk was addressed on an external service owned by your team'
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml]: @BigEasyJ , please remove `TESTING`

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 3]
```diff
@@ -2959,7 +2959,105 @@ tasks:
...
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      htmlBody:
+        simple: |-
+          <!DOCTYPE html>
+          <html lang="en">
+          <body>
+              <p> THIS IS A TEST, PLEASE IGNORE <br><br>
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml]: @BigEasyJ , please remove this blurb as well

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , looks good!  I approve this!

**Comment by daryakoval** [comment]
Please comment here when the pr is ready for merge. Thanks.

**Comment by johnnywilkes** [comment]
@daryakoval , this is ready to merge

**Pull request merged**